modules/top-level/output.nix

```diff
@@ -95,6 +95,13 @@ in
         })
       ) allPlugins;
 
+      # Python3 dependencies from all plugins
+      python3Dependencies =
+        let
+          deps = map (p: p.python3Dependencies or (_: [ ])) allPluginsOverrided;
+        in
+        ps: builtins.concatMap (f: f ps) deps;
+
```

Comment:
Yes. We need to collect all the dependencies and provide them to `makeNeovimConfig`. I did it by attaching dependencies to a plugin pack, so that the following already existing code won't be changed, only the plugin list, but I guess those dependencies can be provided by concating with `extraPython3Packages`.